### PR TITLE
Removed test file dependency from parse int tests

### PR DESCRIPTION
**Pull request by JPricey**
@soul2197

### CODE REVIEW

File: test/utils.js [hunk 1]
```diff
@@ -10,28 +10,34 @@ describe('#convert_byte_array_to_string', () => {
...
-  it('returns the correct value', (done) => {
-    Midi.Utils.xhr('/base/res/mario.midi', (bytes) => {
-      expect(Midi.Utils.parse_int32(bytes, 4)).to.eql(6);
-      expect(Midi.Utils.parse_int32(bytes, 18)).to.eql(19);
-      expect(Midi.Utils.parse_int32(bytes, 45)).to.eql(2457);
-      expect(Midi.Utils.parse_int32(bytes, 2510)).to.eql(2537);
-      expect(Midi.Utils.parse_int32(bytes, 5055)).to.eql(2367);
-      expect(Midi.Utils.parse_int32(bytes, 7430)).to.eql(3064);
+  it('returns the correct value', () => {
+    // Test cases in the form: [[bytes], index, value]
```
Comment (soul2197) [inline test/utils.js]: i would remove index from this test to keep it simple. can test index in a separate test

File: test/utils.js [hunk 2]
```diff
@@ -10,28 +10,34 @@ describe('#convert_byte_array_to_string', () => {
...
 });
 
 describe('parse_int16', () => {
-  it('returns the correct value', (done) => {
-    Midi.Utils.xhr('/base/res/mario.midi', (bytes) => {
-      expect(Midi.Utils.parse_int16(bytes, 8)).to.eql(1);
-      expect(Midi.Utils.parse_int16(bytes, 10)).to.eql(5);
-      expect(Midi.Utils.parse_int16(bytes, 12)).to.eql(96);
+  it('returns the correct value', () => {
+    // Test cases in the form: [[bytes], index, value]
```
Comment (soul2197) [inline test/utils.js]: same

**Comment by soul2197** [comment]
also can rebase and add test for `parse_int24`

**Comment by JPricey** [comment]
Fun facts about JS: Numbers are 64 bit floats with a 53 bit mantissa, so ints up to 53 bits are fully supported.  
BUT, bit operators on integers behave as if the integers are 32 bits, and thus will cause overflow at that point.

That's why I changed the `<< 24` to ` * Math.pow(2, 24)`

**Pull request merged**